pontoon/contributors/urls.py

```diff
@@ -62,6 +62,12 @@ class UsernameConverter(StringConverter):
         views.toggle_user_profile_attribute,
         name="pontoon.contributors.toggle_user_profile_attribute",
     ),
+    # API: Toggle user theme preference
+    path(
+        "api/v1/user/theme/<username:username>/",
```

Comment:
We should instead use `"api/v1/user/<username:username>/theme/"` (user identifier should follow the "user").